templates/katello.yaml.erb

```
@@ -28,12 +28,6 @@
     :url: <%= @pulp_url %>
     :ca_cert_file: <%= @pulp_ca_cert %>
 
-  :use_pulp_2_for_content_type:
-    :docker: <%= @use_pulp_2_for_docker %>
-    :file: <%= @use_pulp_2_for_file %>
-    :yum: <%= @use_pulp_2_for_yum %>
-    :deb: <%= @use_pulp_2_for_deb %>
-
   :katello_applicability: <%= !@use_pulp_2_for_yum %>
```

`@use_pulp_2_for_yum` is now undefined
```suggestion
  :katello_applicability: true
```

Since it'll be true in both branches of `if @pulp2_support`, it can be moved out of it.